pycqed/analysis_v2/base_analysis.py

```diff
@@ -372,7 +386,14 @@ def add_measured_data(raw_data_dict, compression_factor=1):
                     ssp = np.arange(len(ssp) * compression_factor)
                 raw_data_dict['hard_sweep_points'] = hsp
                 raw_data_dict['soft_sweep_points'] = ssp
-
+            elif sp is not None:
+                # deal with hybrid measurements
+                if sweep_points.shape[0] == 1 and len(sp) > 1:
+                    hybrid_measurement = True
+                    hsp = np.arange(len(next(iter(sp[0].values()))[0]))
+                    ssp = np.arange(len(next(iter(sp[1].values()))[0]))
```

Comment:
sp, ssp , sweep_points. I think those variable names are quite confusing. Maybe time to rename sweep_points to "mc_points" here also?

Reply:
Yes! But i will keep sp or? Just because it has a symmetry to cp which we use throughout timedomain_analysis.py

Reply:
yes, I agree, we could keep sp, would rename "sweep_points" to mc_points